Submitted By: Bruce Dubbs <bdubbs at linuxfromscratch dot org>
Date: 2018-03-30
Initial Package Version: 2.89
Upstream Status: Not Submitted
Origin: Self
Description: Remove programs superceeded by other packages: wall, mountpoint, 
             last, lastb, mesg, sulogin, and utmpdump.

Rediffed for vsrsion 2.93.  Also remove readbootlog.

diff -Naur sysvinit-2.93/src/Makefile sysvinit-2.93.new/src/Makefile
--- sysvinit-2.93/src/Makefile	2018-11-24 10:38:12.000000000 -0600
+++ sysvinit-2.93.new/src/Makefile	2018-11-24 11:30:37.483077795 -0600
@@ -24,18 +24,21 @@
 # For some known distributions we do not build all programs, otherwise we do.
 BIN	=
 SBIN	= init halt shutdown runlevel killall5 fstab-decode
-USRBIN	= last mesg readbootlog
+#USRBIN	= last mesg readbootlog
 
-MAN1	= last.1 lastb.1 mesg.1 readbootlog.1
+#MAN1	= last.1 lastb.1 mesg.1 readbootlog.1
 MAN5	= initscript.5 inittab.5 initctl.5
-MAN8	= halt.8 init.8 killall5.8 pidof.8 poweroff.8 reboot.8 runlevel.8
+#MAN8	= halt.8 init.8 killall5.8 pidof.8 poweroff.8 reboot.8 runlevel.8
+MAN8	= halt.8 init.8 killall5.8         poweroff.8 reboot.8 runlevel.8
 MAN8	+= shutdown.8 telinit.8 fstab-decode.8
 
 ifeq ($(DISTRO),)
-SBIN	+= sulogin bootlogd
-USRBIN	+= utmpdump wall
-MAN1	+= utmpdump.1 wall.1
-MAN8	+= sulogin.8 bootlogd.8
+#SBIN	+= sulogin bootlogd
+SBIN	+= bootlogd
+#USRBIN	+= utmpdump wall
+#MAN1	+= utmpdump.1 wall.1
+#MAN8	+= sulogin.8 bootlogd.8
+MAN8	+= bootlogd.8
 endif
 
 ifeq ($(DISTRO),Debian)
@@ -197,10 +200,10 @@
 		ln -sf halt $(ROOT)/sbin/reboot
 		ln -sf halt $(ROOT)/sbin/poweroff
 		ln -sf init $(ROOT)/sbin/telinit
-		ln -sf /sbin/killall5 $(ROOT)/bin/pidof
-		if [ ! -f $(ROOT)/usr/bin/lastb ]; then \
-			ln -sf last $(ROOT)/usr/bin/lastb; \
-		fi
+		#ln -sf /sbin/killall5 $(ROOT)/bin/pidof
+		#if [ ! -f $(ROOT)/usr/bin/lastb ]; then \
+		#	ln -sf last $(ROOT)/usr/bin/lastb; \
+		#fi
 		$(INSTALL_DIR) $(ROOT)/usr/include/
 		$(INSTALL_DATA) initreq.h $(ROOT)/usr/include/
 		$(INSTALL_DIR) $(ROOT)$(MANDIR)/man1/
@@ -218,12 +221,12 @@
 			$(INSTALL_DATA) ../man/$$man $(ROOT)$(MANDIR)/man8/; \
 			sed -i "1{ $(MANDB); }" $(ROOT)$(MANDIR)/man8/$$man ; \
 		done
-ifeq ($(ROOT),)
+#ifeq ($(ROOT),)
 		#
 		# This part is skipped on Debian systems, the
 		# debian.preinst script takes care of it.
-		@if [ ! -p /run/initctl ]; then \
-		 echo "Creating /run/initctl"; \
-		 rm -f /run/initctl; \
-		 mknod -m 600 /run/initctl p; fi
-endif
+		#@if [ ! -p /run/initctl ]; then \
+		# echo "Creating /run/initctl"; \
+		# rm -f /run/initctl; \
+		# mknod -m 600 /run/initctl p; fi
+#endif
